modelscope_agent/llm/custom.py

```diff
@@ -0,0 +1,96 @@
+import os
+from typing import Dict, Iterator, List, Optional
+
+import json
+import requests
+
+from .base import BaseChatModel, register_llm
+
+
+@register_llm('custom')
+class CustomLLM(BaseChatModel):
+    '''
+        the llm service from http.
```

Comment:
make it clear what is the api schema of the backeend service

Reply:
input and output

Reply:
removed the entire customLLM, should be replaced by fastchat or llamaindex later